src/internal/grpcutil/server.go

```diff
@@ -44,6 +46,8 @@ func NewServer(ctx context.Context, publicPortTLSAllowed bool, options ...grpc.S
 			MinTime:             5 * time.Second,
 			PermitWithoutStream: true,
 		}),
+		grpc.ChainUnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
+		grpc.ChainStreamInterceptor(grpc_prometheus.StreamServerInterceptor),
```

Comment:
Multiple `grpc.Chain*Interceptor` options passed to the same NewServer call compose, so it's safe for calling code to add interceptors in addition to these.